integration-tests/utils/integration-helpers.js

```diff
@@ -141,87 +140,17 @@ const withUser = test => async args => {
   }
 }
 
-const addUserToBilling = _.flow(withSignedInPage, withUserToken)(async ({ page, billingProject, email }) => {
-  await page.evaluate((email, billingProject) => {
-    return window.Ajax().Billing.addProjectUser(billingProject, ['User'], email)
-  }, email, billingProject)
-
-  console.info(`added user to: ${billingProject}`)
-
-  const userList = await page.evaluate(billingProject => {
-    return window.Ajax().Billing.listProjectUsers(billingProject)
-  }, billingProject)
-
-  const billingUser = _.find({ email }, userList)
-
-  console.info(`test user was added to the billing project with the role: ${!!billingUser && billingUser.role}`)
-})
-
-const removeUserFromBilling = _.flow(withSignedInPage, withUserToken)(async ({ page, billingProject, email }) => {
-  await page.evaluate((email, billingProject) => {
-    return window.Ajax().Billing.removeProjectUser(billingProject, ['User'], email)
-  }, email, billingProject)
-
-  console.info(`removed user from: ${billingProject}`)
-})
-
-const withBilling = test => async options => {
-  console.log('withBilling ...')
-  await addUserToBilling(options)
-
-  try {
-    await test({ ...options })
-  } finally {
-    console.log('withBilling cleanup ...')
-    await deleteRuntimes(options)
-    await removeUserFromBilling(options)
-  }
-}
-
-const deleteRuntimes = _.flow(withSignedInPage, withUserToken)(async ({ page, billingProject, email }) => {
-  const deletedRuntimes = await page.evaluate(async billingProject => {
-    const runtimes = await window.Ajax().Runtimes.list({ googleProject: billingProject, role: 'creator' })
+const deleteRuntimes = async ({ page, billingProject, workspaceName }) => {
+  const deletedRuntimes = await page.evaluate(async (billingProject, workspaceName) => {
+    const { workspace: { googleProject } } = await window.Ajax().Workspaces.workspace(billingProject, workspaceName).details(['workspace'])
+    const runtimes = await window.Ajax().Runtimes.list({ googleProject, role: 'creator' })
     return Promise.all(_.map(async runtime => {
       await window.Ajax().Runtimes.runtime(runtime.googleProject, runtime.runtimeName).delete(true) // true = also delete persistent disk.
       return runtime.runtimeName
     }, _.remove({ status: 'Deleting' }, runtimes)))
-  }, billingProject, email)
+  }, billingProject, workspaceName)
   console.info(`deleted runtimes: ${deletedRuntimes}`)
-})
-
-const registerUser = withSignedInPage(async ({ page, token }) => {
```

Comment:
is this done somewhere else now?

Reply:
Instead of creating and registering a new user each time, this test now runs as `Scarlett.Flowerpicker@test.firecloud.org`, who is registered in each environment.

Reply:
that's great 👍